skbase/validate/_named_objects.py

```diff
@@ -68,6 +69,10 @@ def is_sequence_named_objects(
           depends on whether `seq_to_check` follows sequence of named
           BaseObject format.
 
+    object_type : class, default=None
```

Comment:
I would say that it can also be a tuple of class, this might be useful, e.g., in pipelines